src/extension/ExtensionContextHandler.ts

```diff
@@ -58,5 +61,19 @@ export class ExtensionContextHandler {
         });
         this.context.subscriptions.push(integrationsTreeView);
         this.context.subscriptions.push(vscode.commands.registerCommand('kaoto.integrations.refresh', () => integrationsProvider.refresh()));
+        this.registerNewCamelYamlFiles();
```

Comment:
picky: I would named it `registerNewCamelYamlFilesCommands` to avoid risk of potential confusion of creating some specific yaml files